modules/apps/product-navigation/product-navigation-taglib/src/main/resources/META-INF/resources/personal_menu/page.jsp

```diff
@@ -82,18 +82,24 @@ if (size != null) {
 	</c:choose>
 
 	<%
-	ResourceURL resourceURL = PortletURLFactoryUtil.create(request, PersonalMenuPortletKeys.PERSONAL_MENU, PortletRequest.RESOURCE_PHASE);
-
-	resourceURL.setParameter("currentURL", themeDisplay.getURLCurrent());
-	resourceURL.setParameter("portletId", themeDisplay.getPpid());
-	resourceURL.setResourceID("/product_navigation_personal_menu/get_personal_menu_items");
+	String resourceURLString = PortletURLBuilder.create(
+		PortletURLFactoryUtil.create(request, PersonalMenuPortletKeys.PERSONAL_MENU, PortletRequest.RESOURCE_PHASE)
+	).setParameter(
+		"currentURL", themeDisplay.getURLCurrent()
+	).setParameter(
+		"p_p_resource_id", "/product_navigation_personal_menu/get_personal_menu_items"
+	).setParameter(
+		"portletId", themeDisplay.getPpid()
+	).setWindowState(
+		LiferayWindowState.EXCLUSIVE
+	).buildString();
```

Comment:
Can you inline this variable? It is used only once.

Reply:
I'll check if the source formatter is good with that change.

Reply:
the source formatter didn't complain